src/snowflake/connector/ssl_wrap_socket.py

```diff
@@ -41,36 +34,6 @@
 """
 FEATURE_OCSP_RESPONSE_CACHE_FILE_NAME = None
 
-
```

Comment:
Are these for python2? I forgot why I added these...

Reply:
I'm not sure, let me dig around!

Reply:
I can't find the reason of why they were added, they were part of the initial release.
But basically this whole commit came from noticing that `ssl_wrap_socket` function was unused and then deleting that exposed a lot of things that were only called from that function.

Reply:
Looks like this removed the dependency of `ssl_wrap_socket` in this module.
https://github.com/snowflakedb/snowflake-connector-python/commit/6c84a997